lib/rubocop/cop/layout/indentation_width.rb

```diff
@@ -179,6 +176,24 @@ def check_members(base, members)
           end
         end
 
+        def select_check_member(member)
+          if member &&
```

Comment:
Perhaps we can use a node pattern to clean this up, e.g.:

```ruby
def_node_matcher :access_modifier?, <<-PATTERN
  [(send ...) access_modifier?]
PATTERN

def select_check_member(member)
  return unless member

  if access_modifier?(member.children.first)
    member.children.first
  else
    member
  end
end
```

Reply:
Oh, I could not think of this method. Thank you for telling me a fine approach!